src/futils.cpp

```diff
@@ -48,6 +49,19 @@
 #include <libproc.h>
 #endif
 
+#if defined(__FreeBSD__)
+#include <sys/mount.h>
+#include <sys/param.h>
+#include <sys/queue.h>
```

Comment:
I am surprised to see so many headers here. It seems like you are not using some of them ... could we get rid of some of them ?

Reply:
Yes.  You're smart.  I cut'n'pasted them from somewhere.  I only need 5 of those.  I submit a change.

I'd like to add a new API: `Exiv2::StringVector Exiv2::getLoadedLibraries()` and put it in src/futils.cpp.  That can be called from src/version.cpp and move a lot of platform dependant code out of src/version.cpp.  Let's add that in v0.28 when the code is ported 'to master'.